static/styles/zulip.css

```diff
@@ -2964,11 +2964,18 @@ div.topic_edit_spinner .loading_indicator_spinner {
 }
 
 #scroll-to-bottom-button-container {
-    display: none;
-    z-index: 3;
+    display: block;
+    z-index: -1;
```

Comment:
Why would you mess with `z-index` when you can animate `visibility` directly?

```css
visibility: hidden;
opacity: 0;
transition: visibility 500ms, opacity 500ms ease-in-out;

&.show {
    visibility: visible;
    opacity: 1;
}
```

Reply:
This is a much better approach. Thanks, Anders!